confluent_kafka/avro/cached_schema_registry_client.py

```diff
@@ -315,6 +316,25 @@ def get_latest_schema(self, subject):
 
         self._cache_schema(schema, schema_id, subject, version)
         return (schema_id, schema, version)
+    
+
+    def get_latest_schema(self, subject):
+        """
+        GET /subjects/(string: subject)/versions/(versionId: version)
```

Comment:
`(versionId: version)` should be 'latest' here

Reply:
why not adding a parameter "version" so we could retrieve the schema given a subject & the verssion?